pkgs/tools/networking/networkmanager/sstp/default.nix

```diff
@@ -20,15 +20,12 @@
 
 stdenv.mkDerivation rec {
   pname = "NetworkManager-sstp";
-  version = "unstable-2023-03-09";
+  version = "1.3.2";
```

Comment:
That version looks older though – according to [NEWS](https://gitlab.gnome.org/GNOME/network-manager-sstp/-/blob/b42de8fbaf21ab6b9a4169aede6bda548490d154/NEWS#L1), it was released on 2022-11-27.

Reply:
Never mind, the changelog is wrong, it was released in 2023: https://gitlab.gnome.org/GNOME/network-manager-sstp/-/tags/release-1.3.2-1